pkg/proto/configuration/bb_worker/bb_worker.proto

```diff
@@ -113,4 +113,8 @@ message RunnerConfiguration {
   // as announced to the scheduler. In addition to the labels specified,
   // one named "thread" is added automatically in case concurrency > 1.
   map<string, string> worker_id = 8;
+
+  // Device nodes to install in input root.
+  // e.g. ["null", "zero", "random"]
+  repeated string devices = 9;
```

Comment:
Nit: Maybe call this `input_root_device_nodes`? `devices` might be a bit too generic.